### PR TITLE
XLS-0073d AMMClawback

### PR DESCRIPTION
**Pull request by yinyiqian1**
the discussion link:  https://github.com/XRPLF/XRPL-Standards/discussions/212
the rippled PR link: https://github.com/XRPLF/rippled/pull/5142

### CODE REVIEW

File: XLS-0074d-amm-clawback/README.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+| `tfClawTwoAssets` | 0x00000001   | Indicates if the issuer wants to claw back both tokens in the pool. The two assets must both issued by the issuer|
+
+- It can be set only when the issuer issues both assets in the AMM pool.
+- If set, both the `Asset` token and the paired toke `Asset2` will be clawed back.
+- If not set, only the `Asset` token will be clawed back, and the paired token `Asset2` will go back to the holder.
+
+#### 2.3.3. AMMClawback transaction examples
+
+##### 2.3.3.1 Only one token is issued by the issuer
+Assue we have an AMM pool consisting two tokens FOO and Bar. And the proportion of FOO and BAR is 1:2. And the issuer `rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL` only issues FOO, BAR is issued by some other account.
```
Comment (gregtatcam) [inline XLS-0074d-amm-clawback/README.md:167]: Typo: Assue -> Assume.

File: XLS-0074d-amm-clawback/README.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+- Upon execution, this transaction enables the issuer `rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL` to claw back at most 1000 FOO owned by holder `rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B` from AMM pool FOO/BAR.
+- The transaction will result in the withdrawal of two corresponding assets from the AMM account on the current proportion:  
+  - The asset issued by the `Account` will be returned to the issuer `Account`. So 1000 FOO will be returned to the issuer `rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL`.
+  - The other asset will be transferred back to the holder's wallet. Since the proportion of FOO and BAR is 1:2, 2000 BAR will be transferred back to the holder's wallet.
+  - If `Amount` is not given or its subfield `value` exceeds the holder's available balance, then all the holder's FOO will be clawed back from the AMM pool.
+- `tfClawTwoAssets` can not be set because BAR is not issued by the issuer.
+
+##### 2.3.3.2 Both tokens are issued by the same issuer
+Assue we have an AMM pool consisting two tokens FOO and Bar. And the proportion of FOO and BAR is 1:2. And the issuer `rPdYxU9dNkbzC5Y2h4jLbVJ3rMRrk7WVRL` issues both FOO and BAR.
```
Comment (gregtatcam) [inline XLS-0074d-amm-clawback/README.md:198]: Typo: Assue -> Assume.

File: XLS-0074d-amm-clawback/README.md (hunk 1 above)
Comment (yinyiqian1) [reply XLS-0074d-amm-clawback/README.md]: updated

File: XLS-0074d-amm-clawback/README.md (hunk 2 above)
Comment (yinyiqian1) [reply XLS-0074d-amm-clawback/README.md]: updated

**Review by SirJosh1987 — APPROVED** [review]

**Review by shawnxie999 — APPROVED** [review]

**Review by dookynme-bit — COMMENTED** [review]
Approve

**Pull request merged**